Multihash ipfs_address
Filepath file
---
bool success
string error_msg